sourcegraph/about
# Remove 4.0 launch banner

Close #5816 - Removes 4.0 launch banner from home page & asset used only in this section

### Note
- Don't merge until Friday 10/07 ❗ 

### Test
1. Ensure prettier has standardized the proposed changes.
2. Observe no breaking changes & absent banner on home page
